nixos/modules/services/x11/desktop-managers/surf-display.nix

```diff
@@ -0,0 +1,88 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.surf-display;
+
+  surfDisplayConf = ''
+    # Surf Kiosk Display: Wrap around surf browser and turn your
+    # system into a browser screen in KIOSK-mode.
+
+    # default download URI for all display screens if not configured individually
+    DEFAULT_WWW_URI="${cfg.defaultWwwUri}"
```

Comment:
This config is pretty simple, are all the things documented anywhere?

I'd recommend if we haven't already, making options for everything that isn't default not set or unneeded and having the `extraConfig` option?

Reply:
It's the default config from their repository, there is no more documentation than the comments in this config as far as I know.

I'm not sure how to define a type for those options where they are an array (the DISPLAYS option), if you know how to do that that would be great :)

Reply:
For `DISPLAYS` it looks a little complicated, not sure what it does.

I'd leave that one up to `extraConfig`.

Reply:
I've pushed a bunch of additional options now, with extraConfig that has displays as example value. I'm not entirely sure how to use those options. But I'm pretty sure that this is good enough :slightly_smiling_face: